app/estimate_square_size.go

```diff
@@ -61,14 +61,26 @@ func estimateCompactShares(squareSize uint64, ptxs []parsedTx) int {
 		txbytes += txLen
 	}
 
-	sharesUsed := 1
-	if txbytes <= appconsts.FirstCompactShareContentSize {
-		return sharesUsed
+	return compactSharesUsed(txbytes)
+}
+
+// compactSharesUsed returns the number of compact shares used to store
+// numBytes.
+func compactSharesUsed(numBytes int) int {
```

Comment:
[very optional suggestion]

perhaps we keep this in the shares packages with `SparseSharesUsed`?

Reply:
Good point! There was already a `compactSharesNeeded` so I opted to de-duplicate functions and export that in https://github.com/celestiaorg/celestia-app/pull/1225/commits/2c0198bebc35fcd199161d2b8a58820fa40a843d